Homework_4/Task_4.py

```diff
@@ -0,0 +1,7 @@
+str1 = str(input("Enter first string: "))
+str2 = str(input("Enter second string: "))
+
+if str1 == str2:
+    print("The strings are equal")
+else:
+    print("The strings are not equal")
```

Comment:
This task can be simplier
str1 = input("Enter first string: ")
str2 = input("Enter second string: ")
result = str1 == str2

print(f'String {str1} and string {str2} have equals result {result}')